src/main/resources/application.yaml

```
@@ -42,3 +42,9 @@ spring:
 azure:
   application-insights:
     instrumentation-key: ${hmc.AppInsightsInstrumentationKey:00000000-0000-0000-0000-000000000000}
+
+jms:
+  servicebus:
+    connection-string: ${HMC_SERVICE_BUS_CONNECTION_STRING:Endpoint=sb://hmc-servicebus-demo.servicebus.windows.net/;SharedAccessKeyName=[KEYNAME];SharedAccessKey=[KEY]}
+    topic-name: ${HMC_SERVICE_BUS_TOPIC:hmc-to-cft-demo}
+    subscription-name: ${HMC_SERVICE_BUS_SUBSCRIPTION:hmc-subs-to-cft-demo}
```

```suggestion
    connection-string: ${HMC_SERVICE_BUS_CONNECTION_STRING}
    topic-name: ${HMC_SERVICE_BUS_TOPIC}
    subscription-name: ${HMC_SERVICE_BUS_SUBSCRIPTION}
```
Only need these changes on local please 🙏

---

changed

---

Re-opening for visibility as think this commit hasn't been pushed yet?